lib/SetupChecks/ModelFilesCheck.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+// SPDX-FileCopyrightText: Anupam Kumar <kyteinsky@gmail.com>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+namespace OCA\SttWhisper\SetupChecks;
+
+use OCA\SttWhisper\Service\DownloadModelsService;
+use OCP\IL10N;
+use OCP\SetupCheck\ISetupCheck;
+use OCP\SetupCheck\SetupResult;
+
+class ModelFilesCheck implements ISetupCheck {
+	public function __construct(
+		private IL10N $l10n,
+	) {
+	}
+
+	public function getCategory(): string {
+		return 'whisper';
+	}
+
+	public function getName(): string {
+		return 'Whisper Speech-To-Text';
```

Comment:
In the original PR in server getName() returns a text like `Checking for XXX of YYY app`